src/main/java/ee/tuleva/onboarding/payment/PaymentProviderCallbackService.java

```diff
@@ -43,6 +43,7 @@ public void processToken(String serializedToken) {
               .amount(amount)
               .internalReference(internalReference.getUuid())
               .user(user)
+              .status(PaymentStatus.PENDING)
```

Comment:
There might be a different status in the JWT token coming from Montonio

Reply:
We should probably double check that the Montonio payment status is `finalized`

Reply:
Done, thanks